scheduler/ecs/ecs.go

```diff
@@ -424,7 +397,83 @@ func (m *ecsProcessManager) updateCreateService(ctx context.Context, app *schedu
 	return m.createService(ctx, app, p)
 }
 
-func (m *ecsProcessManager) Processes(ctx context.Context, appID string) ([]*scheduler.Process, error) {
+func (m *Scheduler) loadBalancer(ctx context.Context, app *scheduler.App, p *scheduler.Process) (string, error) {
```

Comment:
should this just be `createLoadBalancer` then? The name of the method was a little confusing - also, if not, maybe a small comment to describe the point of this?  `loadBalancer` doesn't scream 'this creates something' to me.